bower.json

```diff
@@ -10,9 +10,13 @@
     "lodash": "~4.5.0",
     "perfect-scrollbar": "~0.6.10",
     "pretender": "~0.12.0",
-    "sinonjs": "1.17.1"
+    "sinonjs": "1.17.1",
+    "ember-load-initializers": "0.1.7",
```

Comment:
Line 14 and 15 I think these two packages need to mvoe into the package.json file and remove from here. Use Frost-sort for reference.https://github.com/ciena-frost/ember-frost-sort/blob/master/package.json#L49